packages/network_traffic/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.30.2"
```

```suggestion
- version: "1.31.0"
```
Since it seems this is an enhancement and not a fix. Also the version needs to be changed in the manifest